README.md

```diff
@@ -1,39 +1,76 @@
 # FXChrony_M5STICK_PLUS
 
-FX Chrony display for the M5 Stick C Plus
+## Introduction
 
-## Required Board definitions: 
+This repository contains the FX Chrony display code for the M5 Stick C Plus. The FX Chrony display is a simple and effective way to keep track of your shooting data in real-time. With this project, you can quickly and easily get started with using the M5 Stick C Plus to display important shooting information such as your shot velocity, shot count, and more.
 
-In the Arduino IDE go to File | Preferences and add 
+Instructions to start with Arduino IDE 'https://docs.m5stack.com/en/quick_start/m5stickc/arduino'
 
-https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/arduino/package_m5stack_index.json 
+# Setting up the Arduino IDE environment for M5StickC
 
-To the Additional board manager URLS
+## Step 1: Download and install the Arduino IDE
+1. Download the latest version of the Arduino IDE from the official website: https://www.arduino.cc/en/software
+2. Install the Arduino IDE on your computer
 
-Go to Tools | Board | Boards manager 
+## Step 2: Add the ESP32 board manager URL
```

Comment:
I think this is the URL
https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/arduino/package_m5stack_index.json

I think we need to restart the IDE after adding this